lib/puppet/type/java_ks.rb

```diff
@@ -152,6 +152,13 @@ def self.title_patterns
         ]
       ],
       [
+        /^(.*):([a-z]:(\/|\\).*)$/i,
+        [
+            [ :name, identity ],
+            [ :target, identity ]
+        ]
```

Comment:
Having three matching groups, means that this will probably not do what you want?